xla/backends/gpu/codegen/triton/support_test.cc

```diff
@@ -2416,6 +2476,15 @@ ENTRY triton_computation {
                             data_type)) {
     fail_mode = ExpectedFailMode::kFailOrCrash;
   }
+  if (cc.IsRocm()) {
+    if (absl::c_linear_search(std::vector{F8E4M3FN, F8E5M2FNUZ, F8E4M3FNUZ, F64},
+                                data_type) ||
+        (absl::c_linear_search(std::vector{S64, S32, S16, BF16, F16, F32},
+                                data_type)  &&
+         algorithm == xla::PrecisionConfig::ALG_DOT_F64_F64_F64)) {
+      fail_mode = ExpectedFailMode::kFailOrCrash;
```

Comment:
Why `FailOrCrash`? What happens exactly? We should avoid this mode if possible and prefer something mroe specific.